tests/e2e/scenarios/labels-and-annotations/inventory/host_vars/label-annot-east.yaml

```diff
@@ -0,0 +1,5 @@
+---
+# Kubeconfig path for east site
+kubeconfig_1: "{{ ansible_env.HOME }}/.kube/config"
+kubeconfig: "{{ kubeconfig_1 }}"
```

Comment:
Should it be `kubeconfig_2` here, in order to allow 2 distinct clusters to be used?

Reply:
No, this test was designed to run in a single cluster